src/machine/usb.go

```diff
@@ -212,6 +213,9 @@ func handleStandardSetup(setup usb.Setup) bool {
 		if setup.WValueL == 1 { // DEVICEREMOTEWAKEUP
 			isRemoteWakeUpEnabled = false
 		} else if setup.WValueL == 0 { // ENDPOINTHALT
+			if usbStallHandler[setup.WIndex&0x7F] != nil {
+				return usbStallHandler[setup.WIndex&0x7F](setup)
```

Comment:
`WIndex&0x7f` could use a comment. Will it ever overflow on rogue input?
Also below.

Reply:
Oh, that's a good point. I'll push up a fix in a sec